### PR TITLE
Fix a ton of linter errors, mostly prop types

### PR DESCRIPTION
**Pull request by dannyradden**

### CODE REVIEW

File: client/src/containers/CampaignPage.js [hunk 1]
```diff
@@ -57,6 +57,12 @@ CampaignPage.propTypes = {
   fetchSignatures: PropTypes.func.isRequired,
   params: PropTypes.shape({
     id: PropTypes.string
+  }).isRequired,
+  signature: PropTypes.shape({
+    loaded: PropTypes.bool.isRequired,
+    loading: PropTypes.bool.isRequired,
+    signatures: PropTypes.arrayOf(PropTypes.object),
+    error: PropTypes.objectOf(PropTypes.any)
   }).isRequired
```
Comment (chompasina) [inline client/src/containers/CampaignPage.js]: Great example of why proptypes are good documentation, especially on an app with new folks coming and going. I'm glad we're keeping proptypes for this reason.

**Comment by chompasina** [comment]
@dannyradden looks good to me. I just added one last import line to resolve the last warning. It's so clean now! Yay!

**Comment by chompasina** [comment]
(We think there are issues on production with the callbacks but we don't want to hold this up, so merging in now... FYI @dannyradden )

**Comment by wdoug** [comment]
👍

**Pull request merged**